### DISCUSSION TITLE
Overleaf 中文生僻字问题

### DISCUSSION THREAD

**Discussion by Rui-Alexander-Sun** [SYSU-SCC/sysu-thesis#83] [General]
有时候写论文会遇到中文生僻字，Overleaf 会报错，解决方法：
1. C:\Windows\Fonts
2. 找到 SimSun.ttc
3. 将文件上传至和 main.tex 同一文件夹，即顶端文件夹
4. 在需要输入生僻字的位置，如“喨”，输入“{\CJKfontspec{SimSun.ttc}喨}”，即可显示生僻字，不过和周围的字体看起来会不一样

参考自：
https://blog.csdn.net/m0_54352040/article/details/116113986
https://blog.csdn.net/qq_41860497/article/details/122005706
Mac系统视频教程：https://www.bilibili.com/video/av208015528/

**Comment by note286** [comment]
你可以查看xeCJK手册，直接配置中文字体，例如：

```latex
\setCJKmainfont[AutoFakeBold,AutoFakeSlant]{simsun.ttc}
```

**Reply by Rui-Alexander-Sun** [reply]
感谢！我去试试~
